modules/core/src/main/java/org/apache/ignite/internal/visor/query/VisorQueryCacheDetailMetricsCollectorTask.java

```diff
@@ -37,10 +37,10 @@
 import static org.apache.ignite.internal.processors.cache.GridCacheUtils.isSystemCache;
 
 /**
- * Task to collect cache query metrics.
+ * Collect query detailed metrics from cache context only. For compatibility when node does not have indexing module.
  */
 @GridInternal
-public class VisorQueryDetailMetricsCollectorTask extends VisorMultiNodeTask<VisorQueryDetailMetricsCollectorTaskArg,
+public class VisorQueryCacheDetailMetricsCollectorTask extends VisorMultiNodeTask<VisorQueryDetailMetricsCollectorTaskArg,
```

Comment:
I think it is better to leave old task with old names.
And add new names for new tasks!

Reply:
Old task renamed to old name.